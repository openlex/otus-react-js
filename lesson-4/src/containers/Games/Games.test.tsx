import React from 'react';
import { shallow } from "enzyme";
import { Games } from '@containers';

describe("Games page markup", () => {
	it('match snapshoot', () => {
		const onAddGameMock = jest.fn();
		const gameForm = shallow(<Games/>);

		expect(gameForm).toMatchSnapshot();
	});
});
