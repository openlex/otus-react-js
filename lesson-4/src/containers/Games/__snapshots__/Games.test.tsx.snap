// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Games page markup match snapshoot 1`] = `ShallowWrapper {}`;
